diff --git a/packaging/deploy.py b/packaging/deploy.py
index 29c892a5d..64c5c9ef1 100755
--- a/packaging/deploy.py
+++ b/packaging/deploy.py
@@ -141,6 +141,8 @@ def deploy_windows(qt_path, source_path, build_path, target_path, extra_libs):
 	os.mkdir(target_installer_path)
 	shutil.copy(os.path.join(build_path, 'otter-browser.exe'), target_installer_path)
 	shutil.copy(os.path.join(source_path, 'COPYING'), target_installer_path)
+	print("~~~ COPYING {0} -> {1}".format(os.path.join(source_path, 'COPYING'), target_installer_path))
+	os.system(r"dir C:\Users\travis\build\qtwebkit\release-tests\otter-browser\packaging\otter-browser-packages\input")
 	deploy_locale(source_path, os.path.join(target_path, 'input'))
 	run_command([escape_windows_executable_path(windeployqt_command), os.path.join(target_path, r'input\otter-browser.exe')])
 
@@ -190,7 +192,6 @@ def deploy_windows(qt_path, source_path, build_path, target_path, extra_libs):
 		file.write('--portable')
 
 	run_command(['powershell', 'Compress-Archive', '"{}"'.format(target_release_path), '"{}.zip"'.format(os.path.join(target_path, release_name))])
-	shutil.rmtree(target_release_path)
 
 if __name__ == '__main__':
 	parser = argparse.ArgumentParser(description='Otter Browser deployment tool.')
